packages/falter-berlin-autoupdate/files/autoupdate.sh

```diff
@@ -100,7 +100,7 @@ log "starting autoupdate..."
 ##################
 #  Update-stuff
 
-if echo "$FREIFUNK_RELEASE" | grep "snapshot"; then
+if ! echo "$FREIFUNK_RELEASE" | grep -e '^[0-9]*\.[0-9]*\.[0-9]$'; then
```

Comment:
This regex will match something like `..3` and should be changed to the more specific: `^[0-9]+\.[0-9]+\.[0-9]+$` that will also cover multiple digets in the 3rd part of the version number.